### ISSUE TITLE
Enable ImageCarousel and IframeCarousel on magazine page#1554

### ISSUE THREAD

**Issue by padms** [equinor/energyvision#1555]

**Comment by NickHaggerty1** [comment]
@fernandolucchesi @padms Hey, any chance we can prioritise this one and deploy? Need image carousel for a quite urgent story.
Thanks in advance

**Comment by NickHaggerty1** [comment]
Oh, sorry I see the other card in deploy lane. Ignore me :)

**Comment by SvSven** [comment]
Right, so always actually _run_ the Studio before saying it's good - even with such a stupidly simple thing 👀
